cmd/neofs-cli/modules/container/util.go

```diff
@@ -55,3 +55,7 @@ func getSession(cmd *cobra.Command) *session.Container {
 
 	return &res
 }
+
+func getAwaitContext(cmd *cobra.Command) (context.Context, context.CancelFunc) {
+	return commonflags.GetCommandContextWithAwait(cmd, "await", time.Minute)
```

Comment:
Better now, but I think unless we have it somewhere on
```
$ ./bin/neofs-cli container create --help
Create new container and register it in the NeoFS. 
It will be stored in sidechain when inner ring will accepts it.

Usage:
  neofs-cli container create [flags]

Flags:
      --address string        Address of wallet account
  -a, --attributes strings    Comma separated pairs of container attributes in form of Key1=Value1,Key2=Value2
      --await                 Block execution until container is persisted
      --basic-acl string      HEX encoded basic ACL value or keywords like 'public-read-write', 'private', 'eacl-public-read' (default "private")
      --disable-timestamp     Disable timestamp container attribute
  -f, --force                 Skip placement validity check
  -h, --help                  help for create
      --name string           Container name attribute
  -p, --policy string         QL-encoded or JSON-encoded placement policy or path to file with it
  -r, --rpc-endpoint string   Remote node address (as 'multiaddr' or '<host>:<port>')
      --session string        Filepath to a JSON- or binary-encoded token of the container PUT session
      --subnet string         String representation of container subnetwork
  -t, --timeout duration      Timeout for the operation (default 15s)
      --ttl uint32            TTL value in request meta header (default 2)
  -w, --wallet string         Path to the wallet
  -x, --xhdr strings          Request X-Headers in form of Key=Value

Global Flags:
  -c, --config string   Config file (default is $HOME/.config/neofs-cli/config.yaml)
  -v, --verbose         Verbose output
```
it'd still be very unobvious to users.